src/types/Notification.ts

```diff
@@ -62,7 +62,21 @@ export interface DocumentNotification<
   /**
    * Updated document that triggered the notification
    */
-  result: KDocument<TDocContent>;
+  result: {
+    /**
+     * The message or full document content.
+     */
+    _source: TDocContent;
+    /**
+     * Document unique ID.
+     * `null` if the notification is from a real-time message.
+     */
+    _id: string | null;
+    /**
+     * List of fields that have been updated (only available on document partial updates).
+     */
```

Comment:
Not sure what « only available on document partial updates » means. Is it missing? Is it `null`?

Reply:
I just tested, and the field is missing, so the type is good.